### PR TITLE
AdvancedClover article

### PR DESCRIPTION
**Pull request by maximmasterr**
Статья команды FTL для CopterHack 2021 про advancedClover

### CODE REVIEW

**Comment by okalachev** [comment]
Is this article ready for final editing and merging?

File: docs/ru/advancedClover.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  * Автокомплит
+  * Возможность просмотра топиков
+  * Возможность просмотра топиков с изображениями
+  * Возможность вызова сервисов
+  * Возможность просмотра статуса коптера
+* Библиотека для Python
+  * Упрощение работы с навигацией
+  * Упрощение работы с камерой
+
+![Пример автокомплита, терминала и файлового менеджера ](https://i.imgur.com/9JwgXQH.png)
```
Comment (okalachev) [inline docs/ru/advancedClover.md:18]: Embedding images from external resources is not allowed. Please commit images to the repo (compress with online optimizers, use smaller resolution or jpg if they are too large).

**Review by okalachev — CHANGES_REQUESTED** [review]

**Comment by okalachev** [comment]
Please, put the team name and the contact link (e-mail, telegram, ...) somewhere in the header.

File: docs/ru/advancedClover.md (hunk 1 above)
Comment (maximmasterr) [reply docs/ru/advancedClover.md]: Fixed

**Comment by maximmasterr** [comment]
> Is this article ready for final editing and merging?

Yes

**Comment by okalachev** [comment]
@maximmasterr, I'm performing some edits in your branch, give me please the rights, so I could push them.

**Comment by maximmasterr** [comment]
sent invitation

**Comment by okalachev** [comment]
@maximmasterr, I edited the article a little, is everything ok?

**Comment by maximmasterr** [comment]
Yes

**Review by okalachev — APPROVED** [review]

**Pull request merged**